api/profiles.yaml

```diff
@@ -35,6 +35,13 @@ commands:
     description: Create profile
     method: POST
     path: service/oee-bundle/configurationmanager/2/configuration
+    exampleList:
+      - description: Syntax to create OEE profile
+        command: c8y oee profiles create --deviceId [device-id] --name [oee-profile-name]
```

Comment:
- will this really work without specifying a profile.jsonnet file? Does it use the bodyTemplates.template?
- if we keep this example, please change the description similar to the description of the following example